doc/api/magic-sql.md

```diff
@@ -136,6 +136,21 @@ Or pass an alias (**added in 0.5.2**):
 %sql --close db-two
 ```
 
+
+## Specify creator function
```

Comment:
according to the [docs](https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.create_engine.params.creator), creator must be a function that returns a DBAPI connection:

```
In [11]: engine = create_engine("sqlite://", creator=lambda:sqlite3.connect(""))

In [12]: engine.execute('SELECT name FROM sqlite_schema WHERE type=\'table\' ORDER BY name;')
Out[12]: <sqlalchemy.engine.cursor.LegacyCursorResult at 0x103531840>

In [13]: engine.execute('SELECT name FROM sqlite_schema WHERE type=\'table\' ORDER BY name;').fetchall()
Out[13]: []
```

please update the examples and the test.

Reply:
yeah, I got the same error. probably an issue on duckdb's end, please add an example with sqlite as I showed in the previous comment